src/main/java/service/MenuService.java

```diff
@@ -0,0 +1,86 @@
+package service;
+
+
+import lombok.RequiredArgsConstructor;
+import model.MenuEntry;
+import repository.Repository;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+public class MenuService {
+    private final Repository<MenuEntry> repo;
+    private List<MenuEntry> entries;
+
+
+    public void loadMenu() {
+        System.out.println("loading menu");
+        try {
+            this.entries =  repo.readALl();
+        } catch (IOException e) {
```

Comment:
not sure if io exception should be handled in service, eventually chained exception